### ISSUE TITLE
[A11y] [Issue 3] Need clear button labels within chat

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3133]
# 3 Severity 1 – Need clear button labels within chat

> This is filed to https://github.com/microsoft/BotFramework-WebChat/issues/3133.

## Background

After selecting the customer's location from combo box, new set of buttons appears, that announced as "Yes" and "No", with no other context. If user tries to navigate to the message above those buttons in browse mode, screen reader keeps getting kicked out of browse mode and screen reader users are unable to navigate to that message and read context of those buttons.

### Expected

1. More descriptive labels should be provided for those buttons.
2. Screen reader users should be able to navigate to the message/question "Are you ill, or caring for someone who's ill?" in browse mode.

## Investigation notes

We saw the accessible tree looks correct:

1. `list`
   - This is the transcript
1. `listitem`
1. `group`
   - Whole row of the message, including avatar
1. `generic` "Bot CDC said, Where are you located? . Sent at April 30..."
   - This is the question
1. `generic` x N (these are cosmetics, non-verbal)
1. `generic` "United States Outside the US"
   - This is due to `tabindex="0"`
   - This is the first element in the tree owned by Adaptive Cards renderer
1. `generic`
1. `button` "United States"
1. `generic`
1. `text` "United States"

Need some more time to understand if the `role="group"` is related to `tabindex="0"` or not.

[Bug]
